week-3/alarmclock/index.html

```diff
@@ -4,13 +4,15 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Title here</title>
+    <script defer="alarmclock.js"></script>
+
+    <title>Alarm clock app</title>
   </head>
   <body>
     <div class="centre">
-      <h1 id="timeRemaining">Time Remaining: 00:00</h1>
+      <h1 id="timeRemaining">Time Remaining: <span id="time">00:00</span></h1>
       <label for="alarmSet">Set time to:</label>
-      <input id="alarmSet" type="number" />
+      <input title="Please enter a value lower than 3600." id="alarmSet" type="number" />
```

Comment:
You might have wanted placeholder not title her (or both) https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/placeholder

Reply:
Thanks for the suggestion! I forgot it 🙂  fixed.